invenio_communities/assets/semantic-ui/js/invenio_communities/details_search/components.js

```diff
@@ -187,6 +190,11 @@ export const CommunityRecordSearchAppLayout = ({ config }) => {
 
 CommunityRecordSearchAppLayout.propTypes = {
   config: PropTypes.object.isRequired,
+  appName: PropTypes.string,
+};
+
+CommunityRecordSearchAppLayout.defaultProps = {
+  appName: undefined,
```

Comment:
should we be consistent with defaults? see https://github.com/inveniosoftware/invenio-search-ui/pull/172/files#diff-49ee36867952ffdc6ad4dbf49fbb94ea41a5457f670763f2b3cdad63f8ed7249R82